tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -0,0 +1,143 @@
+---
+- name: Ensure CPO source folder
+  shell:
+    executable: /bin/bash
+    cmd: |
+      rm -rf $GOPATH/src/k8s.io/cloud-provider-openstack
+      mkdir -p $GOPATH/src/k8s.io; cd $_
```

Comment:
we dont need to use GOPATH anymore using gomod we can just clone in home dir

Reply:
I copied from OCCM..  doesn't matter, right?

Reply:
not required, so I prefer to keep it simple instead . But just a suggestion or improvement for later on.